pulp_container/app/migrations/0033_raise_warning_for_repair.py

```diff
@@ -0,0 +1,21 @@
+import warnings
+
+from django.db import migrations
+
+
+def print_warning_for_repair(apps, schema_editor):
+    warnings.warn(
+        "Please, run the 'pulpcore-manager container-repair-media-type' command to repair "
```

Comment:
for possible future needs can you make it  follow similar style as in pulp-rpm? https://github.com/pulp/pulp_rpm/blob/main/pulp_rpm/app/management/commands/rpm-datarepair.py#L28
you would have `pulpcore-manager container-datarepair --issue 1234`

Reply:
And what should be the issue number in this case? 997? I would opt not to specify any issue numbers. Is it user-friendly? Also, in this case, there is no issue number: https://github.com/pulp/pulp_rpm/blob/main/pulp_rpm/app/management/commands/rpm-trim-changelogs.py.

Reply:
yes, 977.

Reply:
you don't want to have 10 management data repair commands in case you have 10 issues. You want to have one interface.

Reply:
if you really want to keep the name as is i am ok with that too, hopefully we won't have many cases of data repair but in case we do then we'd need to write a new command again